addons/beehave/debug/global_debugger.gd

```diff
@@ -13,7 +13,7 @@ func _on_debug_message(message: String, data: Array) -> bool:
 		_set_active_tree(data[0])
 		return true
 	if message == "visibility_changed":
-		if _active_tree:
+		if _active_tree && is_instance_valid(_active_tree):
```

Comment:
Also had to add this because the previously active tree gets chopped when reloading the scene.